stdlib/function.ncl

```diff
@@ -7,5 +7,35 @@
     const : forall a b. a -> b -> a
     | doc "The const function, results in the first argument."
     = fun x y => x,
+
+    compose : forall a b c. (b -> c) -> (a -> b) -> (a -> c)
+    | doc "Function composition, right to left."
+    = fun g f x => x |> f |> g,
+
+    flip : forall a b c. (a -> b -> c) -> b -> a -> c
+    | doc "Flip the argument order for a curried function."
```

Comment:
I find this explanation not extremely clear for someone who doesn't already know what `flip` does.